doc: stabilize --disable-warning (#64742)
nodejs/node: doc/api/cli.md

added:
  - v21.3.0
  - v20.11.0
  changes:
  - version: REPLACEME
    pr-url: https://github.com/nodejs/node/pull/64742

>     description: The `--disable-warning` flag is now stable.

Reviewer: ```suggestion
changes:
  - version: REPLACEME
    pr-url: https://github.com/nodejs/node/pull/64742
    description: The `--disable-warning` flag is now stable.
```

>     description: The `--disable-warning` flags are now stable.

Reviewer 2: ```suggestion
    description: The `--disable-warning` flag is now stable.
```

>   - v20.11.0

Reviewer 3: ```suggestion
  - v20.11.0
  changes:
  - version: REPLACEME
    pr-url: https://github.com/nodejs/node/pull/64742
    description: The `--disable-warning` flags are now stable.
```

-->

> > Stability: 2 - Stable

Reviewer 2: Please add a changes entry (see the YAML comment above)
